pkgs/development/tools/analysis/qcachegrind/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchurl, bash, cmake, qt57, graphviz,  qmakeHook, makeQtWrapper, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  name = "kcachegrind-${version}";
```

Comment:
the name is needed for the url and qcachegrind is a subpackage of kcachegrind for non-kde-environments.